src/transforms/kubernetes/kube_config.rs

```diff
@@ -0,0 +1,91 @@
+#![cfg(feature = "kubernetes-integration-tests")]
+
+use dirs;
+use serde::{Deserialize, Serialize};
+use serde_yaml;
+use std::{fs::File, path::PathBuf};
+
+/// Enviorment variable that can containa path to kubernetes config file.
+const CONFIG_PATH: &str = "KUBECONFIG";
+
+/// Loads configuration from local kubeconfig file, the same
+/// one that kubectl uses.
+pub fn load_kube_config() -> Option<Config> {
```

Comment:
Is there a reason the error context isn't passed up through `ClientConfig::load_kube_config`?

Reply:
Only a minor reason as it's used only for tests so it can be a bit simpler.
Full return should be Result<Option<Config>,Error>

Although, it will probably be used later in regular code, so, yes it would be better to just write it like that now.